meinberlin/static/embed.js

```diff
@@ -28,7 +28,7 @@ $(document).ready(function () {
     }
   })
 
-  $(document).on('submit', function (event) {
+  $(document).on('submit', 'form[action]', function (event) {
```

Comment:
There was a problem with submitting a comment. After submitting it, the embed body was empty because the form had no action attribute. Most likely because it's done with React.